frontend/src/App/App.jsx

```diff
@@ -129,13 +129,14 @@ class App extends React.Component {
             <Route path="/forgot-password" component={ForgotPassword} />
             <Route path="/reset-password" component={ResetPassword} />
             <Route
-              path="/invitations/:token"
+              path="/invitations/:token/:inviteToken?"
```

Comment:
Can we have two separate paths?
`invitations/:token` & `invitations/:token/workspaces/:organizationToken
It would be easy to reason about then.